server/src/domain/user/user.service.ts

```diff
@@ -86,8 +86,8 @@ export class UserService {
     }
 
     if (this.systemConfigService.availableVersion)
-      return { availableVersion: this.systemConfigService.availableVersion, available: true };
-    else return { available: false };
+      return { availableVersion: this.systemConfigService.availableVersion };
+    else return { availableVersion: null };
   }
```

Comment:
Do you need the if here? Can't this whole thing just be `return { availableVersion: this.systemConfigService.availableVersion };`?